package.json

```diff
@@ -92,14 +94,15 @@
   },
   "scripts": {
     "compile-abis": "typechain --target ethers-v5 --out-dir \"src/abis/types\" \"src/abis/**/*.json\"",
-    "postinstall": "yarn compile-abis",
+    "postinstall": "yarn compile-abis && yarn husky install",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "vite",
     "build": "tsc && vite build",
     "serve": "vite preview",
     "test": "vitest run",
     "e2e": "playwright test",
-    "lint": "eslint --ext .tsx,.ts ."
+    "lint": "eslint --ext .tsx,.ts .",
+    "pre-commit": "yarn lint-staged && yarn test"
```

Comment:
I think we can also skip testing and only run the linter @zavelevsky what do you think?

@tiagofilipenunes regarding `yarn lint-staged` I only see a `yarn lint` command for eslint.

Where is lint-staged? Shouldnt it be lint?

And is `eslint --ext .tsx,.ts .` sufficient or do we need to run prettier format in addition to this?

Reply:
I also think we can skip testing, since it happens in our CI

lint-staged will run linting on staged files, modify them, and stage those changes, all before the commit. 

it uses the .lintstagedrc.json, and runs eslint, then runs prettier

Reply:
> @zavelevsky removed testing from pre-commit hook. Looks good now?

yes